.github/workflows/cibuild.yml

```diff
@@ -5,7 +5,7 @@ on: [push]
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I ran into an issue with the CI failing all of the sudden (worked just fine the previous day). After googling, it looks like pinning a version of ubuntu solved the issue with python 3.6 [(link here)](https://github.com/slackapi/bolt-python/pull/776). If this is an issue, lmk and we can look for another work around.

Reply:
Ok, I thought it may be something in GitHub. I clicked around to see if I could find it but couldn't. Thanks!

Reply:
In Settings, Branch protection rules: I changed required from [3.6, 3.7] to [3.7, 3.8, 3.9]